metric/example_test.go

```diff
@@ -76,10 +76,18 @@ func ExampleMeter_asynchronous_multiple() {
 	meter := otel.Meter("go.opentelemetry.io/otel/metric#MultiAsyncExample")
 
 	// This is just a sample of memory stats to record from the Memstats
-	heapAlloc, _ := meter.Int64ObservableUpDownCounter("heapAllocs")
-	gcCount, _ := meter.Int64ObservableCounter("gcCount")
+	heapAlloc, err := meter.Int64ObservableGauge("heapAllocs")
+	if err != nil {
+		fmt.Println("Failed to register gauge for heapAllocs")
+		panic(err)
+	}
+	gcCount, err := meter.Int64ObservableGauge("gcCount")
```

Comment:
This changes all of our examples to be for a gauge. I would prefer we change the pretend measured signal to match the instrument so we can showcase all the instrument types.

Reply:
I have an idea. An http server that counts the requests and an up/down counter for online people.

Reply:
Hey, I updated the example a bit. WDYT now?

Reply:
I think you should put a real web server so people can play around. I also think you should add a real logic to online people. If it turns out too complex, then maybe we should think about another example.

Reply:
Hey, I went through the api specifications https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md
It seems the existing example is fine. Just added error handling. consider merging.

Reply:
Runnable examples should be to the point. They should have the fewest boilerplate possible. I think setting up a web server is too much there.